tests/extraction_tests/test_round_three_extraction.py

```diff
@@ -0,0 +1,181 @@
+    }
+
+    # test with Future high street (extend fixture data)
+    mock_place_extract.Answer[0] = "Future_High_Street_Fund"
+    test_vals_future_high_street = extract_project_lookup(test_project_identifiers, mock_place_extract)
+    assert test_vals_future_high_street == {
+        "Test Project 1": "HS-FAK-01",
+        "Test Project 2": "HS-FAK-01",
+        "Test Project 3": "HS-FAK-01",
+    }
+
+
+@patch("core.extraction.towns_fund.TF_PLACE_NAMES_TO_ORGANISATIONS", {"Fake Town": "Fake Canonical Org"})
+def test_extract_programme(mock_place_extract, mock_programme_lookup):
+    """Test programme info extracted as expected."""
+    test_extracted_programme_df = extract_programme(mock_place_extract, mock_programme_lookup)
+    expected_programme_df = pd.read_csv(resources_assertions / "programme_ref_expected.csv")
+    assert_frame_equal(test_extracted_programme_df, expected_programme_df)
+
+
+@patch("core.extraction.towns_fund.TF_PLACE_NAMES_TO_ORGANISATIONS", {"Fake Town": "Fake Canonical Org"})
+def test_extract_organisation(mock_place_extract):
+    """Test organisations details extracted as expected."""
+    test_extracted_organisation_df = extract_organisation(mock_place_extract)
+    expected_org_df = pd.read_csv(resources_assertions / "organisation_ref_expected.csv")
+    assert_frame_equal(test_extracted_organisation_df, expected_org_df)
+
+
+def test_extract_projects(mock_project_lookup, mock_programme_lookup):
+    """Test projects extracted as expected."""
+    mock_project_admin_tab = pd.read_csv(resources_mocks / "project_admin_sheet_mock.csv")
+    test_extracted_projects_df = extract_project(mock_project_admin_tab, mock_project_lookup, mock_programme_lookup)
+    expected_project_details_df = pd.read_csv(resources_assertions / "project_details_expected.csv")
+    assert_frame_equal(test_extracted_projects_df, expected_project_details_df)
+
+
+# TODO: Add test of whole extract, and run some assertions ie that projects line up as expected between tabs etc.
+
+
+# Test intended only as a local debug tool
+@pytest.mark.skip(reason="currently this is just a pytest/pycharm debug entrypoint for ingest work")
+def test_ingest_towns_fund_template():
+    towns_fund_data = pd.read_excel(
+        "EXAMPLE_TF_Reporting_Template_-_TD_-_Newhaven_-_DDMMYY.xlsx",
+        sheet_name=None,  # extract from all sheets
+    )
+    ingest_towns_fund_data(towns_fund_data)
+
+
+# Test intended only as a local debug tool
+@pytest.mark.skip(reason="currently this is just a pytest/pycharm debug entrypoint for ingest work")
+def test_ingest_round_two_historical():
+    # TODO: currently testing with small subset of data (to allow reasonable debugging speed)
+    round_two_data = pd.read_excel(
+        "Round 2 Reporting - Consolidation (MASTER).xlsx",
+        # "Round 2 Reporting - Consolidation.xlsx",
+        sheet_name=None,  # extract from all sheets
+    )
+    ingest_round_two_data_towns_fund(round_two_data)
```

Comment:
I know that Round 1's integration tested was removed - and that our intention is to potentially remove these tests when they're redundant - but if we're keeping the historical integration test for Round 2 it might be logical to keep the integration test for Round 1 unless we're certain there will be no need to re-ingest or alter the data transformations.

Reply:
Yeah, I changed my mind on removing them as I kept needing to debug the pipeline. I saw R1 had gone, it's probably not the end of the world, and can always add back in if we ever need it